rust/processor/src/models/default_models/signatures.rs

```diff
@@ -91,6 +91,22 @@ impl Signature {
         }
     }
 
+    pub fn get_fee_payer_address(
+        t: &TransactionSignaturePB,
+        transaction_version: i64,
+    ) -> Option<String> {
+        match t.signature.as_ref().unwrap_or_else(|| {
+            tracing::error!(
+                transaction_version = transaction_version,
+                "Transaction signature is missing"
+            );
+            panic!("Transaction signature is missing");
+        }) {
+            SignatureEnum::FeePayer(sig) => Some(standardize_address(&sig.fee_payer_address)),
+            _ => None,
```

Comment:
What does this return for transactions where a separate gas payer is not set?

Reply:
It returns None.

Reply:
Shouldn't it return the sender? Or does it default to the sender before persisting somewhere else